pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -542,6 +545,39 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 		}
 	}
 
+	octaviaVersion, err := getSavedAnnotation(kubeClient, names.KuryrOctaviaVersionAnnotation)
+	if err != nil && !apierrors.IsNotFound(err) {
+		return nil, errors.Wrap(err, "failed to get kuryr-config ConfigMap")
+	}
+
+	maxOctaviaVersion, err := getMaxOctaviaAPIVersion(lbClient)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get max octavia api version")
+	}
+
+	// In case the Kuryr config-map is annotated with an Octavia version different
+	// than the current Octavia version, and older than the version that multiple
+	// listeners becomes available and the Octavia provider is amphora, an Octavia
+	// upgrade happened and UDP listeners are now allowd to be created.
```

Comment:
not allowed

Reply:
I meant `now allowed` as this is the result of the upgrade

Reply:
ups... sorry, I made a typo also in the comment: I mean it should be "now allowed" instead of "now allowd"